apps/client/src/app/core/database/services/roster.service.ts

```diff
@@ -55,6 +56,9 @@ export class RosterService extends FirestoreStorage<Roster> {
               platinumFields: 0
             };
           }
+          if (c.class) {
+            c.className = LostarkClass[c.class].toLowerCase();
+          }
```

Comment:
Please implement this in a pipe as asked in the first review.

The issue with implementing this here is that we would have a computed property that could be saved in database, but will never be updated by the user.

Instead, we should have a `classIcon` pipe that just returns `LostArkClass[classInput].toLowerCase()` in its transform method.

Examples of pipes and how to implement custom ones can be found here: https://angular.io/guide/pipes